None of pip, tox-envlist, and virtualenv are needed to run tests.
tox version pin is too strict.

--- deprecation-alias-0.4.0/tox.ini.orig
+++ deprecation-alias-0.4.0/tox.ini
@@ -39,11 +39,6 @@
     build
 skip_missing_interpreters = True
 isolated_build = True
-requires =
-    pip>=21,!=22.2
-    tox-envlist>=0.2.1
-    tox~=3.0
-    virtualenv!=20.16.0
 
 [envlists]
 test =
